service/history/workflow/cache/cache_test.go

```diff
@@ -368,6 +366,8 @@ func (s *workflowCacheSuite) TestHistoryCacheConcurrentAccess_Release() {
 	release(nil)
 }
 
+/*
```

Comment:
Why not `s.Skip()`?

Reply:
"If was not just failing on fresh main, but also stuck."